src/components/ModalPortal.js

```diff
@@ -321,6 +321,12 @@ export default class ModalPortal extends Component {
       return acc;
     }, {});
 
+  dataAttributes = items =>
```

Comment:
Hi @kaushik94, you can merge both functions `ariaAttributes` and `dataAttributes`. The only difference is the prefix of the key.

Here is one example (naming can be better):

```jsx
{...this.attributesFromObject('aria', this.props.aria || {})}
{...this.attributesFromObject('data', this.props.data || {})}
```